src/sentry/conf/server.py

```diff
@@ -115,6 +115,10 @@ def env(
 # List of IP subnets which should not be accessible
 SENTRY_DISALLOWED_IPS = ()
 
+# List of Region silo IP addresses which are accessible from the control silo.
+# These are individual IP addresses, not subnets (CIDR notation).
+SENTRY_REGION_SILO_IPS = ()
```

Comment:
You should be able to type hint this to avoid the `cast` later on.  Would we be able to generate this value from the REGION_CONFIG? so that we don't need to have the same IP lists in multiple places? Or would this IP list have larger ranges than the region configuration would?

I think we'll also need to define this configuration when the split silo devserver is being used.